cppForSwig/Makefile.am

```diff
@@ -0,0 +1,120 @@
+LINK = $(CXX)
+
+PYTHON_INCLUDES=@PYTHON_INCLUDES@
+PYTHON_LIBS=@PYTHON_LIBS@
+
+if TARGET_LINUX
+UNAME=Linux
+else
+if TARGET_WINDOWS
+UNAME=Windows
+else
+if TARGET_DARWIN
+UNAME=Darwin
+endif
+endif
+endif
+
+# Export variables to make them available to cryptopp Makefile.
+export AR
+export CXX
+export RANLIB
+export UNAME
+
+if TARGET_LINUX
+CRYPTOPP_EXT=.a
+else
+if TARGET_WINDOWS
+CRYPTOPP_EXT=.a
```

Comment:
Yes. I originally thought there would be different extensions for the different targets, so that is why I started doing that, but it turns out that it is `.a` for all platforms. I could just put a `CRYPTOPP_EXT=.a` line instead if you want.